buildFatJar fails since upgrade to gradle 7.2

```
./gradlew buildFatJar
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.2/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 
'

FAILURE: Build failed with an exception.

* Where:
Build file '/apps/was/jenkins/workspace/Compile_CruiseControl/cruise-control/build.gradle' line: 311

* What went wrong:
Could not determine the dependencies of task ':cruise-control:buildFatJar'.
> Could not get unknown property 'compile' for configuration container of type org.gradle.api.internal.artifacts.configurations.DefaultConfigurationContainer.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 10s
5 actionable tasks: 5 up-to-date
```

--- a/build.gradle
+++ b/build.gradle
@@ -324,9 +324,10 @@ project(':cruise-control') {
     duplicatesStrategy 'exclude'
   }
 
-  tasks.create(name: "buildFatJar", type: Jar) {
+  tasks.create(name: "buildFatJar", type: Jar, dependsOn: ["createVersionFile", ":cruise-control-metrics-reporter:jar", ":cruise-control-core:jar"]) {
     archiveBaseName = project.name + '-all'
-    from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+    from { configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) } }
     with jar
   }
 

